doc/source/whatsnew/v0.22.0.txt

```diff
@@ -84,7 +84,7 @@ Deprecations
 ~~~~~~~~~~~~
 
 - ``Series.from_array`` and ``SparseSeries.from_array`` are deprecated. Use the normal constructor ``Series(..)`` and ``SparseSeries(..)`` instead (:issue:`18213`).
--
+- ``Series.asobject``, ``DatetimeIndex.asobject`` and ``PeriodIndex.asobject``have been deprecated. Use '.astype(object).values' instead (:issue:`18477`)
```

Comment:
its just ``.astype(object)``

Reply:
It you follow the call chain down it translates to ``ser._data.blocks[0].values.astype(object)``. So I'd say it should be "use ``.values.astype(object)``", which is an ndarray, while ``.astype(object)`` returns a Series.

I'll use the form``.values.astype(object)``, unless you comment